### PR TITLE
doc: Updated nodejs stack readme

### PR DESCRIPTION
**Pull request by neeraj-laad**
Updated readme for nodejs stack

### CODE REVIEW

File: incubator/nodejs/README.md [hunk 1]
```diff
@@ -1,56 +1,36 @@
 # Node.js Stack
 
-A stack is the foundation of your application. This stack provides a prototype platform definition for Node.js v10 based projects.
+The Node.js stack provides a consistent way of developing [Node.js](https://nodejs.org/) applications. As an asynchronous event-driven JavaScript runtime, Node is designed to build scalable network applications.
 
-## Templates
+This stack is based on `Node.js v10` runtime and allows you to bring any Node.js application that can work with this version of the runtime.
```
Comment (helenmasters) [inline incubator/nodejs/README.md]: Can you add "the" in front of `Node.js v10`?
What do we mean by the word 'bring'?

File: incubator/nodejs/README.md [hunk 2]
```diff
@@ -1,56 +1,36 @@
...
 
-A template is your starting ppoint to begin developing your application.
+## Templates
 
-### simple template
+Templates are used to create your local project and start your development. When initializing your project you will be provided with the simplest Node.js application you can possibly write.
```
Comment (helenmasters) [inline incubator/nodejs/README.md]: "you will be provided with the simplest Node.js application you can possibly write"
then we say "The default template provides a simple application that logs a message to the console"
I found this confusing as I wondered to start with if these were two different things

Why don't you merge sentences together, as I presume that simple Node.js app is created with the default template?

File: incubator/nodejs/README.md [hunk 3]
```diff
@@ -1,56 +1,36 @@
...
+
+1. After your project has been initialized you can then run your application using the following command:
+
+    ```bash
+    appsody run
+    ```
+
+    This launches a Docker container that continuously re-builds and re-runs your project, exposing it on port 3000.
+
+    You can continue to edit the application in your preferred IDE (VSCode or others) and your changes will be reflected in the running container within a few seconds.
```
Comment (helenmasters) [inline incubator/nodejs/README.md]: (VSCode or other)

File: incubator/nodejs/README.md (hunk 1 above)
Comment (neeraj-laad) [reply incubator/nodejs/README.md]: > addressed

File: incubator/nodejs/README.md (hunk 2 above)
Comment (neeraj-laad) [reply incubator/nodejs/README.md]: addressed

File: incubator/nodejs/README.md (hunk 3 above)
Comment (neeraj-laad) [reply incubator/nodejs/README.md]: addressed

**Pull request merged**